official/vision/image_classification/dataset_factory.py

```diff
@@ -477,7 +477,7 @@ def parse_record(self, record: tf.Tensor) -> Tuple[tf.Tensor, tf.Tensor]:
     parsed = tf.io.parse_single_example(record, keys_to_features)
 
     label = tf.reshape(parsed['image/class/label'], shape=[1])
-    label = tf.cast(label, dtype=tf.int32)
+    label = tf.cast(label, tf.int32)
```

Comment:
We should prefer keeping the cast within `preprocess` rather than `parse_record` since `preprocess` is called in all pipelines.

Reply:
fixed